helm/loki/values.yaml

```
@@ -179,12 +179,12 @@ loki:
       #customWriteUrl: http://loki-multi-tenant-proxy.default.svc.cluster.local:3101
       #customBackendUrl: http://loki-multi-tenant-proxy.default.svc.cluster.local:3102
     autoscaling:
-      enabled: true
-      minReplicas: 2
+      enabled: false
+      minReplica: 2
```

Beware, that's not the same key as before your change.
```suggestion
      minReplicas: 2
```

Hint: you can review your changes in your PR to spot this kind of errors.